packages/fx-core/src/plugins/solution/fx-solution/debug/util/tasksNext.ts

```diff
@@ -14,18 +14,20 @@ export function generateTasks(
 ): Record<string, unknown>[] {
   /**
    * Referenced by launch.json
-   *   - Pre Debug Check
-   *   - Start Frontend
-   *   - Start Backend
-   *   - Start Bot
+   *   - Pre Debug Check & Start All
    *
    * Referenced inside tasks.json
    *   - validate local prerequisites
    *   - start ngrok
    *   - prepare local environment
+   *   - start all
```

Comment:
I suggest keep the original Case, for customization feature as well as users already familiar with our toolkit.

Reply:
Updated.